Documentation/ReleaseNotes.md

```diff
@@ -0,0 +1,285 @@
+# Microsoft Mixed Reality Toolkit Release Notes
+
+- [Version 2.0.0](#version-200)
+
+## Version 2.0.0
+
+- [Upgrading projects](#upgrading-projects-to-200)
+- [What's new](#whats-new-in-200)
+- [Known issues](#known-issues-in-200)
+
+This release of the Microsoft Mixed Reality Toolkit 2.0.0 supports the following devices and platforms.
+
+- Microsoft HoloLens 2
+- Microsoft HoloLens (1st gen)
+- Windows Mixed Reality Immersive headsets
+- OpenVR
+
+The following software is required.
+
+- Microsoft Visual Studio (2017 or 2019) Community Edition or higher
+- Windows 10 SDK 18362 or later (installed by the Visual Studio Installer)
+- Unity 2018.4, 2019.1 or 2019.2
+
+### Upgrading projects to 2.0.0
+
+Since the RC2 release, there have been several changes that may impact your application projects,
+including some files moving to new folder locations. For the smoothest upgrade path in your
+projects, please use the following steps.
+
+1. Close Unity
+1. Delete all **MixedRealityToolkit** (you may not have all listed folders)
+    - MixedRealityToolkit
+    - MixedRealityToolkit.Examples
+    - MixedRealityToolkit.Extensions
+    - MixedRealityToolkit.Providers
+    - MixedRealityToolkit.SDK
+    - MixedRealityToolkit.Services
+    - MixedRealityToolkit.Tools
+1. Delete your **Library** folder
+1. Re-open your project in Unity
+1. Import the new unity packages
+    - Foundation - _Import this package first_
+    - (Optional) Tools
+    - (Optional) Extensions,
+    - (Optional) Examples are optional)
+1. For each scene in your project
+    - Delete **MixedRealityToolkit** and **MixedRealityPlayspace**, if present, from the Hierarchy
+    - Select **MixedRealityToolkit -> Add to Scene and Configure**
+
+> [!Important]
+> Some profiles have been changed (properties have been added) in 2.0.0. If you have created custom
+profiles, please open them to verify that all of the updated properties are correctly configured.
+
+### What's new in 2.0.0
+
+- [Default HoloLens (1st gen) profile](#default-hololens-1st-gen-profile)
```

Comment:
There are now more paragraphs that don't match the links. Needs to be updated, or removed. Actually the list is pretty long and I don't think people will get that much value in the TOC (they will probably do a once-over read and move on), so I'd recommend just removing the whole TOC.

Reply:
i can do that

Reply:
in progress